README.md

```diff
@@ -56,3 +56,4 @@ git-subtree. See those directories' README files for more details.
 * [XRP Ledger Dev Portal](https://xrpl.org/)
 * [Setup and Installation](https://xrpl.org/install-rippled.html)
 * [Source Documentation (Doxygen)](https://ripple.github.io/rippled)
+* [Learn more about the XRP Ledger](https://www.youtube.com/playlist?list=PLJQ55Tj1hIVZtJ_JdTvSum2qMTsedWkNi)
```

Comment:
```suggestion
* [Learn more about the XRP Ledger (YouTube)](https://www.youtube.com/playlist?list=PLJQ55Tj1hIVZtJ_JdTvSum2qMTsedWkNi)
```
Think it may be helpful to indicate `(YouTube)` so folks know at a glance what to expect if they click the link

Reply:
Wait, sorry but what's the point here? YouTube is widely known and used by many on a daily basis, I don't think clicking on a link and seeing YouTube popup is a surprising/unwanted thing.

Reply:
I can'd hover over a link on mobile for example.